### DISCUSSION TITLE
Folow

### DISCUSSION THREAD

**Discussion by moham313** [community/community#102019] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

hi can you folow me?

**Comment by B0K1NG** [comment]
Hey, will do that! Everyone starts somewhere!

**Comment by queenofcorgis** [comment]
Hi @moham313 , we’re glad you’re here 😀

Thanks for sharing your profile with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
